### DISCUSSION TITLE
Are multiple range requests supported ?

### DISCUSSION THREAD

**Discussion by boxerab** [seaweedfs/seaweedfs#6135] [General]
Hello! I would like to request multiple parts of a file over HTTP using multiple range requests:

i.e.  "Range: bytes=0-1023, 2000-3000"

Is this supported ?

Edit: I do see [here](https://github.com/seaweedfs/seaweedfs/issues/1403#issuecomment-664156987) that they are supported - just want to double check.

Thanks!

**Comment by chrislusf** [comment]
yes

**Reply by boxerab** [reply]
Excellent! Thanks for the confirmation.
